test/torch/linalg/test_operations.py

```diff
@@ -115,3 +115,16 @@ def test_qr_mpc(hook, workers):
 
     # Check if QR == t
     assert ((Q @ R - t).abs() < 1e-2).all()
+
+
+def test_remote_random_number_generation(hook, workers):
```

Comment:
While we’re here, could you add a test for all of them? In practice you should be able to parameterize it

Reply:
As separate functions or all within `test_remote_random_number_generation`?

Reply:
In the same function is good :)

Reply:
Have added tests for the rest of the functions.

Reply:
@robert-wagner I realize now my last commit is not doing what was asked. The problem is that [you can't do remote tensor generation](https://github.com/OpenMined/PySyft/issues/2678) at the moment. So the ones I added are just executed locally despite being done so in the sent model.